client/src/helpers/APICalls/createOrEditMeetups.ts

```diff
@@ -36,9 +35,35 @@ export const createNewMeetup = async (
     }));
 };
 
+export const editMyMeetup = async (
+  meetupId: string,
+  organizerId: string,
+  location: string,
+  locationAddress: string,
+  locationCityStateZip: string,
+  name: string,
+  date: Date,
+  timeStart: string,
+  timeEnd: string,
+  description: string,
+): Promise<AuthApiData> => {
+  const newData = { location, locationAddress, locationCityStateZip, name, date, timeStart, timeEnd, description };
+  const fetchOptions: FetchOptions = {
+    method: 'PUT',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify({ newData, meetupId, organizerId }),
+    credentials: 'include',
+  };
+  return await fetch(`/meetup/update`, fetchOptions)
+    .then((res) => res.json())
+    .catch(() => ({
+      error: { message: 'Unable to connect to server. Please try again' },
+    }));
```

Comment:
Technically, we don't need to do `.then` on this here, I would just await the response from fetch in a variable then to the `.json()` on the data responded back. This maintains good readability and consistency.